Source/Scenes/SubMenus/TASSettingsMenu.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Celeste64.Source.Scenes.SubMenus
```

Comment:
Despite what your IDE might suggest you, remove the `Source` from the namespace.